### ISSUE TITLE
Default state for Pull Requests

### ISSUE THREAD

**Issue by walliski** [rauhryan/huboard#588]
Hi

Since a lot of people use Pull requests in their workflow, it would be nice to have a setting for having a default value for the pull requests. For example if I have the following workflow:

0 - Backlog
1 - Working
2 - Review
3 - Done

Then all the created pull requests would have a default label set as "2 - Review" and would show up in the review column automatically, instead of backlog as they do now.

Is this something that can be done somehow now? Else it would be a nice thing to have..

**Comment by rauhryan** [comment]
> Is this something that can be done somehow now? 

No, but it's a really good idea and we will look into it. 

Thanks for the suggestion

**Comment by benjamincharity** [comment]
Really like this idea.

**Comment by discorick** [comment]
:+1:

**Comment by walliski** [comment]
How's it going? This is the only thing preventing me from using this tool effectively with peer reviewing processes -_-

**Comment by discorick** [comment]
@walliski I was thinking we could accomplish the fairly easily by assuming the second to last column is also the review column - would something like that work for you ?

**Comment by rauhryan** [comment]
Sounds doable to me

**Comment by benjamincharity** [comment]
@rauhryan assuming a column position seems very limiting.

I understand that there are agile 'best practices' that many may be concerned with. However, one of the beautiful parts of Huboard (thus far) is that we can define our own custom flow. Of all of the agile/agile-ish flows that I've seen, they are _very_ rarely ever identical.

- - -

For instance, I have personally worked at three different companies that each had a differing column/review process.

**Company A**: [backlog] - [sprint] - [in progress] - [review] - [ready] - [live]

This was pretty straight forward, yet they would have to kill one of the last two columns for a column based PR approach.

**Company B**: [research] - [backlog] - [sprint] - [in progress] - [done]

This company didn't have a dedicated review column.. rather when a PR was created, it was given the 'in progress' label and assigned to the reviewer.

**Company C**: [goals] - [needs discussion] - [backlog] - [sprint] - [in progress] - [dev review] - [ops review] - [done]

This company has a double review process - first at the code level, then at more of a QA level.

- - -

Long story-short.. different teams have different needs and the beauty of this product is flexibility. 

Maybe we could define the label we want associated with a new PR in the settings panel?

**Comment by patdolem** [comment]
I think the user should be able to define what is the default column for the PRs. For example we have a [InProgress] column as part of our workflow and it should make sense for us to use it as the default one for the PRs instead of the [Backlog] one.

**Comment by discorick** [comment]
@patdolem @benjamincharity Ok, good feedback - it seems like the second-last column assumption is not going to be a viable solution. I think something like a drop-down box to select a default column for PR's in our settings page would be most flexible (rather than use a label)

**Comment by patdolem** [comment]
I agree, a drop-down box in the settings page seems to be a good choice.

**Comment by vine77** [comment]
This sounds like Overv.io's feature to "Automatically advance issues fixed by Pull Requests to: <choose label>" and is a highly desired feature for out team.
